spec/unit/view_spec.rb

```diff
@@ -54,6 +54,15 @@
       expect(v.group.count).to eq mh.group("kde-frameworks").count
       expect(v.group.first.name).to eq mh.group("kde-frameworks").first.name
     end
+
+    it "returns topic" do
+      mh = ManifestHandler.new settings
+      mh.read_remote
+      v = View.new mh
+
+      expect(v.topic("API").count).to eq (mh.topic("API").count)
+      expect(v.topic("API").first.name).to eq (mh.topic("API").first.name)
```

Comment:
Better be explicit here and don't rely on the ManifestHandler to return the right thing. Just compare to the literal value you expect.